### DISCUSSION TITLE
Reviewers for mastodon/documentation repo

### DISCUSSION THREAD

**Discussion by liamstojanovic** [mastodon/mastodon#26331] [Q&A]
I've created a pull request for the 'mastodon/documentation' repository but am unsure how to proceed with review. Who should I contact for assistance? Are there guidelines for contributions to the repo I can reference? Thank you!

PR: [https://github.com/mastodon/documentation/pull/1268](https://github.com/mastodon/documentation/pull/1268)

**Comment by mjankowski** [comment]
Looks like your docs PR was merged.

The actual answer to your question is ... no one knows. Sort of a mystery as to if/when anything gets a response, but there are lightweight guidelines in docs readme: https://github.com/mastodon/documentation?tab=readme-ov-file#contributing-to-the-mastodon-documentation
